### PR TITLE
Avoid escaping semicolon in azure

### PR DESCRIPTION
**Pull request by tandrup**
Fixes Issue #229 

Please let me know if I need to do anything additional

### CODE REVIEW

**Comment by silverl** [comment]
Could we get a code review and merge here please?

File: octodns/provider/azuredns.py [hunk 1]
```diff
@@ -18,6 +20,16 @@
 from .base import BaseProvider
 
 
+def escape_semicolon(s):
+    assert s
+    return string.replace(s, ';', '\\;')
```
[the remark this answers is not in the document]
Comment (tandrup) [reply octodns/provider/azuredns.py]: I just copied this from the `rackspace.py` file. But I can change it. So you want it to be `s.replace(';', '\\;')`?

[the remark this answers is not in the document]
Comment (tandrup) [reply octodns/provider/azuredns.py]: Sorry for the stupid question. I'm not really a python developer

[the remark this answers is not in the document]
Comment (ross) [reply octodns/provider/azuredns.py]: Yeah, was just thinking `s.replace...`. Guess that slipped by without my noticing it over in the rackspace one. It's not a problem, just an unnecessary import and overkill method. If you don't mind switching things over I'll get it merged.

**Comment by silverl** [comment]
Could we get a merge/build here? Thanks

File: octodns/provider/azuredns.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tandrup) [reply octodns/provider/azuredns.py]: I've updated the PR to reflect your changes.

**Comment by ross** [comment]
Thanks!

**Comment by silverl** [comment]
Thanks here as well!
May I ask how long until we see this available in pip?

**Comment by ross** [comment]
There's some large-ish improvements in flight at the moment that I'm planning to land before cutting another release. 

In the meantime you can grab it from pip with a specific sha:

```
-e git+https://git@github.com/github/octodns.git@<sha>#egg=octodns
`

**Comment by silverl** [comment]
Great, thank you very much for that syntax.

**Pull request merged**